src/injectdll/cfuncs.py

```diff
@@ -9,26 +39,27 @@
 from ctypes import windll
 from ctypes.wintypes import BOOL, DWORD, HANDLE, LPVOID, LPCVOID
 
-# C:/PROGRA~1/MICROS~4/VC98/Include/winbase.h 223
-class SECURITY_ATTRIBUTES(Structure):
+
+class SecurityAttributes(Structure):
```

Comment:
Such structures are defined in `<windows.h>` or dependent headers. That's why it looks like in C code. Please keep such names as is to simplify matching them between Python and C. :)